knowledge-base/scheduler-kb-context-menu-on-appointments-and-empty-slots.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Manage Appointments via Context Menu in the Scheduler
```

Comment:
The word "manage" is not used by the customer in their ticket. Consider another word that is more likely to be used by customers in a Google search. For example:

```suggestion
title: Show Context Menu on Scheduler Appointments
```